5.0/en/0x03-Using-ASVS.md

```diff
@@ -1,72 +1,152 @@
+
+Similarly, while the application is responsible for how it consumes input and produces output, if an external process interacts with the application or its data, it is considered out of scope for ASVS. For instance, backing up the application or its data is usually the responsibility of an external process and is not controlled by the application or its developers.
+
+### Security
+
+Every requirement must have a demonstrable impact on security. The absence of a requirement must result in a less secure application, and implementing the requirement must reduce either the likelihood or the impact of a security risk.
 
-The ASVS defines three security verification levels, with each level increasing in depth and complexity. Each ASVS level indicates the security requirements that are required to achieve that level (with the others remaining as recommendations). The general aim is that organizations will start with the lowest level and then move up to the higher levels from there.
+All other considerations, such as functional aspects, code style, or policy requirements, are out of scope.
 
-### Level evaluation criteria
+### Verification
 
-The approach to level definition for version 5.0, was decided after much discussion within the Working Group based on feedback from ASVS users and guided by the version 5.0 principles discussed in the previous chapter. Version 5.0 makes a priority-based evaluation of each requirement based on experience implementing security requirements. Taking this approach, each requirement was evaluated using the following criteria.
+The requirement must be verifiable, and the verification must result in a "fail" or "pass" decision.
 
-#### Risk Reduction
+### Standard
 
-To what extent does the requirement reduce security risk within the application? This takes into account the classic Confidentiality, Integrity, and Availability impact factors as well as considering whether this is a first layer of defense or would be considered defense in depth.
+The ASVS is designed to be a collection of security requirements to be implemented to comply with the standard. This means that requirements are limited to defining the security goal to achieve that. Other related information can be built on top of ASVS or linked via mappings.
 
-In general, the requirements which bring the greatest risk reduction would be in Level 1 or Level 2 and requirements which are still valuable but are either defense in depth or relate to a more niche area or issue are in Level 3.
+Specifically, OWASP has many projects, and the ASVS deliberately avoids overlapping with the content in other projects. For example, developers may have a question, "how do I implement a particular requirement in my particular technology or environment," and this should be covered by the Cheat Sheet Series project. Verifiers may have a question "how do I test this requirement in this environment", and this should be covered by the Web Security Testing Guide project.
```

Comment:
Swap colon and quote?

→ "how do I implement a particular requirement in my particular technology or environment",